expirationd.lua

```diff
@@ -445,10 +445,9 @@ end
 --     False (default) to process each tuple as a single transaction and true
 --     to process tuples from each batch in a single transaction.
 -- @boolean[opt] options.force
---     By default expirationd should process tasks only on the writeable
---     instance, it means that expirationd will not start task processing on a
---     replica. Here the word 'replica' means an instance with at least one
---     configured upstream, it's an option `box.cfg.replication_source`
+--     By default expirationd does not start task processing on a replica. Here
+--     the word 'replica' means an instance with at least one configured
+--     upstream, it's an option `box.cfg.replication_source`
```

Comment:
>     1. Change `box.cfg.replication_source` to `box.cfg.replication` and resolve [something is wrong in documentation #96](https://github.com/tarantool/expirationd/issues/96).

I made such commit, but then I noticed [that we still support Tarantool 1.6.8 (at least in the package requirements)](https://github.com/tarantool/expirationd/pull/109#issuecomment-1141941505). So I decided not to make unnecessary changes in the pull request and just to fix small problems with the current logic. #96 can be fixed with #42.

>     2. Don't introduce a local term 'replica' (which is not exactly same as the usual meaning on this term), but just describe what is going on.
Thank you, done:
```patch
+--     By default expirationd does not start task processing on an instance with
+--     configured upstreams (see [`box.cfg.replication_source`][1] or
+--     [`box.cfg.replication`][2] for Tarantool 1.7.6+). Set the option
+--     to `true` to enable task processing on the instance.
+--
+--     [1]: https://www.tarantool.io/en/doc/1.6/reference/configuration/#cfg-replication-replication-source.
+--     [2]: https://www.tarantool.io/en/doc/latest/reference/configuration/#cfg-replication-replication.
+--
```

>     3. Add a note somewhere near to the top of the API documentation page (https://tarantool.github.io/expirationd/) regarding `force` option. Otherwise only ones who read the whole option list will know how to work with the module.
I've added a note to the `expirationd.start` description:
```patch
+-- NOTE: By default expirationd does not start tasks on an instance with
+-- configured upstreams, see the `force` option.
```

Reply:
1. I would proceed here this way: keep support `replication_source` as the alias for `replication`, but remove it from the documentation.

Reply:
I'm sorry for the misunderstanding. No problem, done.

Reply:
The last nit: the second commit should close #96.

Reply:
Thanks. I've updated the commit message.